### ISSUE TITLE
quirks/websites-with-shared-credential-backends.json should be removed

### ISSUE THREAD

**Issue by erynofwales** [apple/password-manager-resources#825]
quirks/websites-with-shared-credential-backends.json is a legacy data file that has been replaced by quirks/shared-credentials.json. We should remove it.

**Comment by rmondello** [comment]
Yeah, I think it's time. My plan for this is to change the script that converts to the legacy format to take a required output path, then remove the legacy file. Then any password manager who still wants to use the legacy format can use the script on their side, instead of making all contributors do the conversion.

How does this sound? @Cldfire @bradcush

**Comment by bradcush** [comment]
Yea that makes sense, I'm for it. Also thinking out loud that It could be beneficial adding a few small test cases to the CI at some point later since there will no longer be any trace of an artifact in the repository to manually check against. Just to make sure it's working given any kind of updates and it can also serve as a doc for the old format.

**Comment by Cldfire** [comment]
@rmondello that sounds good to me!

(Do note however that I no longer work at 1Password, so I can't speak for them.)

**Comment by rmondello** [comment]
@Cldfire Oh gosh, I'm so embarrassed! My apologies! (But thank you for chiming in regardless.)

**Comment by Cldfire** [comment]
Oh no worries at all! I just wanted to make sure I clarified 😄
